client/src/components/OnboardUser.jsx

```diff
@@ -340,6 +342,15 @@ const OnboardUser = props => {
             SAVE MY RESPONSES
           </Button>
         </div>
+        <Button
+          onClick={() => {
+            setuserWantsToDelete(true);
+          }}
+          className={classes.saveButton}
+        >
+          DELETE ALL OBSERVATION DATA
+        </Button>
```

Comment:
Maybe we should make this button red to make sure users are hyper-aware of this?